vATIS.Desktop/Ui/ViewModels/AtisStationViewModel.cs

```diff
@@ -373,6 +377,13 @@ public AtisStationViewModel(AtisStation station, INetworkConnectionFactory conne
         this.WhenAnyValue(x => x.NetworkConnectionStatus).Skip(1).Subscribe(HandleNetworkStatusChanged);
     }
 
+    private void HandleSetAtisLetter(char letter)
+    {
+        if (letter < 'A' || letter > 'Z')
```

Comment:
```suggestion
        if (letter < mAtisStation.CodeRange.Low || letter > mAtisStation.CodeRange.High)
```

Needs to validate against the defined code range.